connector_prestashop_catalog_manager/consumer.py

```diff
@@ -154,11 +163,12 @@ def product_template_write(session, model_name, record_id, fields):
         return
     model = session.env[model_name]
     record = model.browse(record_id)
-    for binding in record.prestashop_bind_ids:
-        export_record.delay(
-            session, 'prestashop.product.template', binding.id, fields,
-            priority=20,
-        )
+    if set(fields.keys()) - set(EXCLUDE_TEMPLATE_IMAGE_FIELDS):
```

Comment:
A product and all its combinations are exported several times when modifying its main image:

![jobs](https://user-images.githubusercontent.com/39995504/46286747-8d2fb580-c580-11e8-80e9-2ff21ad4311e.png)

This is due to the design of the modules `product` and `product_multi_image`.

These product exports can be avoided and the image will also be exported.